test/benchmark/avro-upsert/metric-verifier/wait_metric

```diff
@@ -13,20 +13,50 @@ import argparse
 import dateparser
 import datetime
 import io
+import math
+import os
 import pathlib
 import sys
 import time
 import typing
 import urllib
+import uuid
 
+import confluent_kafka.admin
 import confluent_kafka.avro
+import confluent_kafka.schema_registry
 import prometheus_api_client
 import prometheus_api_client.utils
-import random_name
 import requests
 
 
-SCHEMAS_DIRECTORY = "/usr/share/benchmarks/schemas"
+METRICS_DIRECTORY = "/usr/share/benchmarks/schemas"
+
+
+def create_topics(args: argparse.Namespace) -> None:
+    """Create the topics for each metric and exit."""
+
+    csr_config = {'url': f'http://{args.csr_host}:{args.csr_port}'}
+    csr = confluent_kafka.schema_registry.SchemaRegistryClient(csr_config)
+
+    kafka_config = {'bootstrap.servers': f"{args.kafka_host}:{args.kafka_port}"}
```

Comment:
micronit: this is the only string in this group that uses double quotes. This wouldn't be an issue if this file was formatted/checked by `black`, suggesting maybe it should be?

Reply:
I'm going to do this as another PR because the change to existing code is non-trivial.